backend/lcfs/db/models/fuel/EndUseType.py

```diff
@@ -29,3 +29,9 @@ class EndUseType(BaseModel, Auditable, DisplayOrder):
         secondary=final_supply_intended_use_association,
         back_populates="intended_use_types",
     )
+    # Keep only one relationship
```

Comment:
What does this comment refer to? I see lots of relationships.

Reply:
Thanks for asking, removing this comment. I forgot to remove it when using useList=False for one-to-one relationship in my first implementation. and did not remove the comment